modules/removing-custom-config-from-cluster.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * rosa_cluster_admin/rosa-configuring-pid-limits.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="removing-custom-config-from-cluster_{context}"]
+= Removing custom configuration from a cluster
```

Comment:
@bhardesty For now we can be more specific but I do anticipate us being able to configure more things this way in future based on discussions with PM, so went generic where possible. I'll review again, but lmk what you reckon.

Reply:
Considering future bookmark and redirect possibilities, I think it's best to go with the generic option here because we can, but I've tried to be a little more specific in the intro sentence to give a better idea of what's going on.

Reply:
+1 I had the same thought at the time